frontend/src/scenes/instance/Licenses/licenseLogic.ts

```diff
@@ -11,7 +11,11 @@ export function isLicenseExpired(license: LicenseType): boolean {
 }
 
 /** The higher the plan, the higher its sorting value - sync with back-end License model */
-const PLAN_TO_SORTING_VALUE: Record<LicensePlan, number> = { [LicensePlan.Scale]: 10, [LicensePlan.Enterprise]: 20 }
+const PLAN_TO_SORTING_VALUE: Record<LicensePlan, number> = {
+    [LicensePlan.Starter]: 0,
```

Comment:
This was because I was following type errors around after I extended the list of AvailableFeatures. But I realized I was dumb and could just use a `Partial<>` for the minimum features and avoid needing to do this.